app/lib/package/backend.dart

```diff
@@ -135,6 +135,7 @@ class PackageBackend {
   Future<models.PackageVersion> lookupPackageVersion(
```

Comment:
Please update documentation comment with why and when it returns `null`.

Whenever, I'm calling something that gets a value (but the value might not be there), it's awesome to know if it'll throw or return `null`.

Reply:
(just a nit)